packages/react-components/services/ReactNativeLogger.ts

```diff
@@ -30,16 +30,21 @@ export default class ReactNativeLogger {
     }
   }
 
+  // TODO: remove this method from here and use our alert banner
```

Comment:
No, there are a few cases when the design calls for a toast message instead of the blue banner. If iOS doesn't have an equivalent, maybe we'll need to make a generic toast component (or change the designs, which I'm personally open to)

Reply:
iOS doesn't have a native equivalent. I'll update the comment and will check with Corey.